src/sentry/static/sentry/app/views/userFeedback/organizationUserFeedback.jsx

```diff
@@ -113,24 +66,58 @@ class OrganizationUserFeedback extends React.Component {
     return children;
   }
 
-  render() {
-    const {status} = getQuery(this.props.location.search);
+  renderList() {
+    if (this.state.reportList.length === 0) {
+      return this.renderEmpty();
+    }
+
+    return this.renderResults();
+  }
+
+  renderEmpty() {
+    return (
+      <EmptyStateWarning>
+        <p>{t('Sorry, no results match your serch query.')}</p>
+      </EmptyStateWarning>
+    );
+  }
+
+  renderNoAccess() {
     return (
       <Content>
-        <Feature
-          features={['organizations:sentry10']}
-          organization={this.props.organization}
-          renderDisabled={this.renderNoAccess}
-        >
+        <Alert type="warning">{t("You don't have access to this feature")}</Alert>
+      </Content>
+    );
+  }
+
+  renderBody() {
+    const {organization, location, params} = this.props;
+    const {status} = getQuery(location.search);
+    const {reportList, reportListPageLinks} = this.state;
+
+    return (
+      <Feature
+        features={['organizations:sentry10']}
+        organization={organization}
+        renderDisabled={this.renderNoAccess}
+      >
+        <GlobalSelectionHeader
+          organization={organization}
+          projects={organization.projects.filter(project => project.isMember)}
```

Comment:
We should probably just make this the default behavior.

Reply:
I agree, will follow up